app/views/includes/head.njk

```diff
@@ -14,7 +14,13 @@
   ga('set', 'displayFeaturesTask', null);
   ga('set', 'transport', 'beacon');
 
-  ga('send', 'pageview');
+  var url = new URL(document.location.href);
+  var search_params = new URLSearchParams(url.search);
+  search_params.delete('email');
+  url.search = search_params.toString();
+  var filtered_page_path = url.pathname + url.search;
+  ga('set', 'page', filtered_page_path);
```

Comment:
Assuming `ga('set', 'page', filtered_page_path);` is equivalent to `ga('send', 'pageview');`

Then this looks good to me

Reply:
Yes this is what the guy from Merkle suggested. I'll forward you the email.